packages/plugin-image-generation/src/environment.ts

```diff
@@ -20,12 +21,13 @@ export const imageGenEnvSchema = z
                 data.HEURIST_API_KEY ||
                 data.FAL_API_KEY ||
                 data.OPENAI_API_KEY ||
-                data.VENICE_API_KEY
+                data.VENICE_API_KEY ||
+                data.LIVEPEER_GATEWAY_URL
             );
         },
         {
             message:
-                "At least one of ANTHROPIC_API_KEY, NINETEEN_AI_API_KEY, TOGETHER_API_KEY, HEURIST_API_KEY, FAL_API_KEY, OPENAI_API_KEY or VENICE_API_KEY is required",
```

Comment:
probably didn't mean to remove NINETEEN_AI_API_KEY

Reply:
thanks, key was added again on the message